### PR TITLE
releases table

### PR DESCRIPTION
**Pull request by leecalcote**
Publishing new Releases page in Meshery docs.

Signed-off-by: leecalcote <leecalcote@gmail.com>

### CODE REVIEW

File: docs/_sass/custom/custom.scss [hunk 1]
```diff
@@ -175,4 +177,7 @@ nav {
 .main-content a:hover {
     font-weight: normal !important;
     color: blue !important;
-}
\ No newline at end of file
+} 
+ul.navigation-list {
+    background-color: aliceblue;
+  }
```
Comment (venilnoronha) [inline docs/_sass/custom/custom.scss]: nit: extra leading spaces.

File: docs/_sass/custom/custom.scss [hunk 2]
```diff
@@ -175,4 +177,7 @@ nav {
 .main-content a:hover {
     font-weight: normal !important;
     color: blue !important;
-}
\ No newline at end of file
+} 
+ul.navigation-list {
```
Comment (venilnoronha) [inline docs/_sass/custom/custom.scss]: nit: missing line break.

File: docs/assets/css/materialize.css [hunk 3]
```diff
@@ -3105,6 +3106,9 @@ td, th {
   vertical-align: middle;
   border-radius: 2px;
 }
+td.centered, th.centered {
```
Comment (venilnoronha) [inline docs/assets/css/materialize.css]: nit: missing line break.

File: docs/installation/releases.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+<table class="responsive-table hover striped">
+  <tr>
+    <th class="centered">Version</th>
+    <th>Description</th>
+    <th>Release Date</th>
+  </tr>
+  <tr>
+    <td class="centered">0.1.0</td>
+    <td>Initial version of Meshery. Connects to Kubernetes and run preconfigured commands on Kubernetes with Istio</td>
```
Comment (venilnoronha) [inline docs/installation/releases.md]: s/Connects/Connect

File: docs/installation/releases.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.2.0</td>
+    <td>Ability to support running custom yaml on kubernetes with istio</td>
```
Comment (venilnoronha) [inline docs/installation/releases.md]: Properly capitalize Kubernetes and Istio.

File: docs/installation/releases.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.3.0</td>
+    <td>Initial version with support of Meshery adapters and release of a pre-alpha version of Istio adapter</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.4.0</td>
+    <td>Meshery's User interface completely re-written using Reactjs and the backend updated as needed.</td>
```
Comment (venilnoronha) [inline docs/installation/releases.md]: s/User/user

File: docs/installation/releases.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+    <td>June 25th, 2019</td>
+  </tr>
+  <tr>
+    <td class="centered">0.11.0</td>
+    <td>Migrate from Configure Meshery to Settings page.</td>
+    <td>June 26th, 2019</td>
+  </tr>
+  <tr>
+    <td class="centered">0.12.0</td>
+    <td>Displace chartjs with billboardjs</td>
```
Comment (venilnoronha) [inline docs/installation/releases.md]: Capitalize framework names.

File: docs/installation/releases.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+    <td></td>
+  </tr>
+  <tr>
+    <td class="centered">0.13.0</td>
+    <td>Migrate from browser local storage to volume / filesystem storage</td>
```
Comment (venilnoronha) [inline docs/installation/releases.md]: End all description elements with a period for consistency.

Comment (leecalcote) [reply docs/installation/releases.md]: Agree, yes.

File: docs/_sass/custom/custom.scss (hunk 1 above)
Comment (leecalcote) [reply docs/_sass/custom/custom.scss]: Good one. Consistency = readability.

File: docs/_sass/custom/custom.scss (hunk 2 above)
Comment (leecalcote) [reply docs/_sass/custom/custom.scss]: Fixed

File: docs/assets/css/materialize.css (hunk 3 above)
Comment (leecalcote) [reply docs/assets/css/materialize.css]: fixed

**Pull request merged**